### ISSUE TITLE
Cannot parse CREATE TABLE with primary key constraint

### ISSUE THREAD

**Issue by sergio91pt** [sbdchd/squawk#70]
The following example taken from [postgres documentation](https://www.postgresql.org/docs/9.6/ddl-constraints.html#DDL-CONSTRAINTS-PRIMARY-KEYS) fails to parse
```sql
CREATE TABLE example (
    a integer,
    b integer,
    c integer,
    PRIMARY KEY (a, c)
);
```

Error:
```
Err(
    CheckSQL(
        ParsingSQL(
            JsonParse(
                "unknown variant `Constraint`, expected `ColumnDef` at line 1 column 770",
            ),
        ),
    ),
)
```

**Comment by sbdchd** [comment]
Thanks for the report, looking into this.


```json
❯ echo "CREATE TABLE example ( a integer, b integer, c integer, PRIMARY KEY (a, c) );" | squawk --dump-ast=raw | jq .
[
  {
    "RawStmt": {
      "stmt": {
        "CreateStmt": {
          "oncommit": 0,
          "relation": {
            "RangeVar": {
              "inh": true,
              "location": 13,
              "relname": "example",
              "relpersistence": "p"
            }
          },
          "tableElts": [
            {
              "ColumnDef": {
                "colname": "a",
                "is_local": true,
                "location": 23,
                "typeName": {
                  "TypeName": {
                    "location": 25,
                    "names": [
                      {
                        "String": {
                          "str": "pg_catalog"
                        }
                      },
                      {
                        "String": {
                          "str": "int4"
                        }
                      }
                    ],
                    "typemod": -1
                  }
                }
              }
            },
            {
              "ColumnDef": {
                "colname": "b",
                "is_local": true,
                "location": 34,
                "typeName": {
                  "TypeName": {
                    "location": 36,
                    "names": [
                      {
                        "String": {
                          "str": "pg_catalog"
                        }
                      },
                      {
                        "String": {
                          "str": "int4"
                        }
                      }
                    ],
                    "typemod": -1
                  }
                }
              }
            },
            {
              "ColumnDef": {
                "colname": "c",
                "is_local": true,
                "location": 45,
                "typeName": {
                  "TypeName": {
                    "location": 47,
                    "names": [
                      {
                        "String": {
                          "str": "pg_catalog"
                        }
                      },
                      {
                        "String": {
                          "str": "int4"
                        }
                      }
                    ],
                    "typemod": -1
                  }
                }
              }
            },
            {
              "Constraint": {
                "contype": 5,
                "keys": [
                  {
                    "String": {
                      "str": "a"
                    }
                  },
                  {
                    "String": {
                      "str": "c"
                    }
                  }
                ],
                "location": 56
              }
            }
          ]
        }
      },
      "stmt_len": 76
    }
  }
]
```

**Comment by sbdchd** [comment]
Should be fixed in `v0.5.2`

Thanks!
